test/upgrade/continual/source-config.toml

```diff
@@ -0,0 +1,9 @@
+# logLevel = 'DEBUG'
+[sender]
+interval = {{ .Config.Interval.Nanoseconds }}
+  [sender.address]
```

Comment:
is this formatted correctly ?

Reply:
Yes. It's advised to indent the part if you are setting some key as object. Here, I'm setting address with `{bootstrapServers: '', topicName: ''}`.

See example: https://github.com/toml-lang/toml#example

Reply:
weird :-)